### PR TITLE
giant clean up of demo pages

### PR DESCRIPTION
**Pull request by notwaldorf**
This is a giant PR and I'm really sorry, so here's a cute: https://media.giphy.com/media/14uXQbPS73Y3qU/giphy.gif

This:
- stops using `paper-styles.html` and uses the specific imports instead
- brings the demo elements to the future, and uses the Polymer 1.1.0 style recommendations
- as part of that, moves the CSS import to a style import
- there was a 'doc' demo that didn't get included in the links, but looks relevant, so i added it too
- the `demo/index.html` is a real page now and not just a headless list of links:
screen shot 2015-11-16 at 2 39 52 pm

### CODE REVIEW

**Comment by addyosmani** [comment]
Initially LGTM. I'm glad to see the 1.1 style recommendations now used here. Also demo page finally has some proper styling. Yay :tada:

File: demo/grid/animated-grid.html [hunk 1]
```diff
@@ -7,70 +7,68 @@
...
 <link rel="import" href="../../../polymer/polymer.html">
-<link rel="import" href="../../../paper-styles/paper-styles.html">
+<link rel="import" href="../../../paper-styles/typography.html">
+<link rel="import" href="../../../iron-flex-layout/iron-flex-layout.html">
 <link rel="import" href="../../neon-shared-element-animatable-behavior.html">
+<link rel="import" href="../shared-styles.html">
 
 <dom-module id="animated-grid">
+  <template>
+    <style include="shared-styles"></style>
```
Comment (cdata) [inline demo/grid/animated-grid.html]: :beers:

File: demo/grid/animated-grid.html [hunk 2]
```diff
@@ -7,70 +7,68 @@
...
+      }
 
-    .tile:nth-of-type(5) {
-      width: calc(100% / 3);
-      height: calc(100% / 3 * 2);
-    }
+      .tile:nth-of-type(1) {
+        width: calc(100% / 3);
+        height: calc(100% / 3 * 2);
+      }
```
Comment (cdata) [inline demo/grid/animated-grid.html]: :100:

File: demo/grid/index.html [hunk 3]
```diff
@@ -17,38 +17,31 @@
...
     <link rel="import" href="../../neon-animations.html">
     <link rel="import" href="animated-grid.html">
     <link rel="import" href="fullsize-page-with-card.html">
 
-    <style>
-
+    <style is="custom-style">
       body {
         overflow: hidden;
+        @apply(--layout-fullbleed);
```
Comment (cdata) [inline demo/grid/index.html]: :dancer:

File: demo/load/animated-grid.html [hunk 4]
```diff
@@ -7,70 +7,68 @@
...
+      }
 
-    .tile:nth-of-type(5) {
-      width: calc(100% / 3);
-      height: calc(100% / 3 * 2);
-    }
+      .tile:nth-of-type(1) {
+        width: calc(100% / 3);
+        height: calc(100% / 3 * 2);
+      }
```
Comment (cdata) [inline demo/load/animated-grid.html]: :100:

File: demo/reprojection/animated-grid.html [hunk 5]
```diff
@@ -7,70 +7,70 @@
...
-      float: left;
-      vertical-align: top;
-      width: calc(100% / 6);
-      height: calc(100% / 3);
+      .tile {
+        display: inline-block;
+        float: left;
+        vertical-align: top;
+        width: calc(100% / 6);
+        height: calc(100% / 3);
```
Comment (cdata) [inline demo/reprojection/animated-grid.html]: :+1:

File: demo/reprojection/reprojected-pages.html [hunk 6]
```diff
@@ -11,15 +11,17 @@
 <link rel="import" href="../../neon-animated-pages.html">
 
 <dom-module id="reprojected-pages">
-
   <template>
-
+    <style>
+      neon-animated-pages {
+        height: 100%;
+      }
```
Comment (cdata) [inline demo/reprojection/reprojected-pages.html]: :+1: Nice catch!

File: demo/tiles/index.html [hunk 7]
```diff
@@ -17,44 +17,36 @@
...
 
-    </style>
-
-    <style is="custom-style">
-
       :root {
-        --color-one: #4dd0e1;
-        --color-two: #ff9800;
+        --color-one: var(--paper-cyan-300);
+        --color-two: var(--paper-orange-500);
```
Comment (cdata) [inline demo/tiles/index.html]: :beers:

**Comment by cdata** [comment]
Very nice cleanup, LGTM :+1:

**Pull request merged**